rpc/src/eth_rpc.rs

```diff
@@ -0,0 +1,30 @@
+use crate::jsonrpsee::core::{async_trait, RpcResult};
+use ethereum_types::U256;
+use ethportal_api::EthApiServer;
+
+pub struct EthApi;
```

Comment:
I think we will need to wrap the history send channel to have access to the history network for future `eth_*` endpoints, same as https://github.com/ethereum/trin/blob/8e97046679b32f419d398d58ac4dd8d60804ceaf/rpc/src/history_rpc.rs#L23

But I understand that this is not necessary the scope of this PR.

Reply:
iiuc, whatever "constructs" `eth_*` responses will need access to every subnetwork. It's out of scope for this pr, but imo should be taken in consideration as we build out this functionality.

Reply:
👍🏻  yeah I'll add access as we go.

I think it could be a good idea to have Options for the networks, so that if someone starts with just history, then we can still serve getBlock calls. Then if they try an `eth_call`, the rpc server will respond with an error saying that you need to re-launch `trin` with the state network enabled.